ansible/playbooks/ptf_installation.yaml

```diff
@@ -5,21 +5,43 @@
   become: true
   become_user: root
   vars:
-    ptf_url: "your_ptf_url"
-    ptf_user: "your_username_here"
-    ptf_password: "your_pass_here"
+    url_timeout: 30
+    url_retries_cnt: 5
+    url_retries_delay: 10
     ptf_dir: "/tmp/ptf_dir"
+  
   tasks:
+    - name: Include external variables
+      ansible.builtin.include_vars: ./vars/hana_media.yaml
+
     - name: Create directory for PTF installation
       file:
         path: "{{ ptf_dir }}"
         state: directory
         mode: '0755'
 
-    - name: Download PTF files recursively with wget
```

Comment:
is it possible to find a way not to trough away this part and to let have both to survive in the same playbooks (or two playbooks if needed)

Reply:
I just did it with 2 playbooks. I think it can be done in 1 by checking which vars are available, but it's also fine like this. What do you think?

Reply:
considering number of tasks in common I'd prefer 1 configurable playbook:
- If user provide `ptf_user,ptf_password,ptf_url` is manual
- If user provide `ptf_file` is the other